### ISSUE TITLE
Crashing Related to Fireballs?

### ISSUE THREAD

**Issue by gusphan** [KozGit/DOOM-3-BFG-VR#303]
I have the VR mod plus all-in-on + ultimate HD texture mod.  I was able to play the first few chapters without issue (other than the loading new map problem).  However, I think it's the 3rd or 4th chapter, but now I'm getting frequent crashes.  I believe it happens when an imp tries to throw a fireball.  Odd thing is, it's not all the time.  But now, I'm lucky if I can play more than 10 minutes without the issue happening.  Any thoughts?

**Comment by KozGit** [comment]
This seems to somehow be related to some of the hi-def texture packs, but unfortunately I haven't had the time yet to identify what the issue is.  It's on the list...

**Comment by Kitano123** [comment]
I crash sometimes when imps throw fireballs but im not using a texture pack.

**Comment by phobos2077** [comment]
The same issue.. but for me the crash is not random but happens every time, so I'm stuck :(
Enpro Plant - there's an Imp near a big round room with an elevator at the back. Game crashes 100% when it throws a fireball. The same happens if I try to avoid him with noclip and use the elevator... or if I just try to run past him with ai_think 0.. there's something bad with this particular room.

**PS:** I solved my problem by completely reinstalling the game, deleting savegame folder, reinstalling just the game and BFG VR w/o any additional crap. And re-playing everything from the beginning... yeah, not the best solution, but hey - the game is fun! I still got occasional crashes often when Imp throw fireballs. But at least now they are not repeated all the time and I was able to finish that damn level finally.

**TL;DR:** Don't use any "HD texture" crap with this mod, kids! It's bad for your health!

**Comment by NPi2Loup** [comment]
Which version did you use ?
Did you try my release ? It may fix these bugs

**Comment by phobos2077** [comment]
@NPi2Loup yes of course I used your version, it doesn't fix my crashes.

**Comment by dfmlege** [comment]
I identified the possible culprit here: #371
